app/scripts/templates/marketing_snippet_ios.mustache

```diff
@@ -4,9 +4,9 @@
         <p>
           {{#t}}Get the full Sync experience.{{/t}}
           <br>
-          {{#t}}Download Firefox for iOS.{{/t}}
+          {{#t}}Firefox is coming soon to iOS!{{/t}}
         </p>
-        <a class="marketing-link marketing-link-ios" href="{{ appStoreLinkDirect }}" target="_blank"><img src="{{ appStoreImage }}" alt="{{#t}}Download Firefox for iOS.{{/t}}"></a>
+        <a class="marketing-link marketing-link-ios" href="https://www.mozilla.org/en-US/newsletter/ios/">{{#t}}Sign up to learn more{{/t}}</a>
```

Comment:
You can probably drop the `/en-US/` from that URL and Mozilla will intelligently redirect users to the correct locale based on their current language.